website/docs/r/service_dynamic_snippet_content_v1.html.markdown

```diff
@@ -128,34 +127,34 @@ resource "fastly_service_dynamic_snippet_content_v1" "my_dyn_content" {
     ignore_changes = [content, ]
   }
 }
- ```
+```
 
 
- ## Argument Reference
+## Argument Reference
 
- The following arguments are supported:
+The following arguments are supported:
 
 * `service_id` - (Required) The ID of the service that the dynamic snippet belongs to
 * `snippet_id` - (Required) The ID of the dynamic snippet that the content belong to
 * `content` - (Required) The VCL code that specifies exactly what the snippet does.
 
- ## Attributes Reference
+## Attributes Reference
 
 * [fastly-vcl](https://docs.fastly.com/api/config#vcl)
 * [fastly-vcl-snippets](https://docs.fastly.com/api/config#snippet)
 
- ## Import
+## Import
 
 This is an example of the import command being applied to the resource named `fastly_service_dynamic_snippet_content_v1.content`
 The resource ID is a combined value of the `service_id` and `snippet_id` separated by a forward slash.
 
- ```
+```
 $ terraform import fastly_service_dynamic_snippet_content_v1.content xxxxxxxxxxxxxxxxxxxx/xxxxxxxxxxxxxxxxxxxx
 ```
 
-If Terraform is already managing remote content against a resource being imported then the user will be asked to remove it from the existing Terraform state.  
+If Terraform is already managing remote content against a resource being imported then the user will be asked to remove it from the existing Terraform state.
 The following is an example of the Terraform state command to remove the resource named fastly_service_dynamic_snippet_content_v1.content` from the Terraform state file.
```

Comment:
The following value is not rendering like the rest of the references on the page.  The value is missing a start backtick, but it does have the closing backtick.

fastly_service_dynamic_snippet_content_v1.content

Reply:
👍 Fixed